src/repr/bdd.rs

```diff
@@ -59,24 +134,34 @@ impl BddPtr {
     ///
     /// Panics if the pointer is constant (i.e., true or false)
     pub fn into_node(&self) -> BddNode {
```

Comment:
actually, I probably need to profile / inline this to make sure there isn't a performance hit.